curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/6400a55edf5e733b47fb7c46.md

```diff
@@ -0,0 +1,158 @@
+---
+id: 6400a55edf5e733b47fb7c46
+title: Step 18
+challengeType: 0
+dashedName: step-18
+---
+
+# --description--
+
+We need to show the the link to the author's author page. You can do this with the `url` you destructured from the array to create a link. When you click the link, the author's page will load up.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here